apps/mobile/src/screens/Onboarding/OnboardingRecommendedUsers.tsx

```diff
@@ -0,0 +1,201 @@
+    followAllRecommendedUsers();
+    setTimeout(async () => await handleNext(), 800);
+  }, [user, followAllRecommendedUsers, handleNext]);
+
+  const handleBack = useCallback(() => {
+    navigation.goBack();
+  }, [navigation]);
+
+  const { top } = useSafeAreaInsets();
+
+  const handleLoadMore = useCallback(() => {
+    if (hasPrevious) {
+      loadPrevious(USERS_PER_PAGE);
+    }
+  }, [hasPrevious, loadPrevious]);
+
+  const recommendedUsers = useMemo(() => {
+    const list = followingPagination.viewer?.suggestedUsers?.edges?.map((edge) => edge?.node) ?? [];
+    return removeNullValues(list);
+  }, [followingPagination.viewer?.suggestedUsers?.edges]);
+
+  const [hasFollowedSomeone, setHasFollowedSomeone] = useState(false);
+
+  return (
+    <View style={{ paddingTop: top }} className="bg-white">
+      <View className="flex flex-col flex-grow space-y-4 px-4 bg-white">
+        <View className="relative flex-row items-center justify-between ">
+          <BackButton onPress={handleBack} />
+          <GalleryTouchableOpacity
+            onPress={handleNext}
+            className="flex flex-row items-center space-x-2"
+            eventElementId="Next button on onboarding recommended users screen"
+            eventName="Next button on onboarding recommended users screen"
+            eventContext={contexts.Onboarding}
+          >
+            <Typography
+              className="text-sm text-metal"
+              font={{ family: 'ABCDiatype', weight: 'Regular' }}
+            >
+              {hasFollowedSomeone ? 'Next' : 'Skip'}
+            </Typography>
+            <RightArrowIcon color={colors.metal} />
+          </GalleryTouchableOpacity>
+        </View>
+
+        <View className="flex flex-col">
+          <Typography className="text-md" font={{ family: 'ABCDiatype', weight: 'Medium' }}>
+            Recommended collectors for you
+          </Typography>
+          <Typography
+            className="text-md text-shadow"
+            font={{ family: 'ABCDiatype', weight: 'Medium' }}
+          >
+            Popular users on gallery
+          </Typography>
+        </View>
+
+        <View
+          style={{
+            height: heightOfUserList,
+          }}
+        >
+          <Suspense fallback={<UserFollowListFallback />}>
+            <UserFollowList
+              onLoadMore={handleLoadMore}
+              userRefs={recommendedUsers}
+              queryRef={followingPagination}
+              onUserPress={noop}
+              onFollowPress={() => setHasFollowedSomeone(true)}
+              isPresentational={true}
+            />
+          </Suspense>
+        </View>
+
+        <View className="flex justify-end mb-8">
+          <Button
+            onPress={handleFollowAll}
+            variant="primary"
+            size="md"
+            className="w-full"
+            eventElementId="Next button on onboarding recommended screen"
+            eventName="Next button on onboarding recommended screen"
```

Comment:
Might be to rename the event name and id here. Seems like this button action will follow all the users